src/js/components/Data/stories/Complex.js

```diff
@@ -10,19 +10,32 @@ const data = [
     name: 'Alpha',
     location: { city: 'Athens', country: 'Greece' },
     economy: { GDP: 100 },
+    colors: ['White', 'Blue'],
   },
   {
     id: 2,
     name: 'Beta',
     location: { city: 'Bangkok', country: 'Thailand' },
     economy: { GDP: 150 },
+    colors: ['Red', 'White', 'Blue'],
+  },
+  {
+    id: 3,
+    name: 'Theta',
+    location: { city: 'Berlin', country: 'Germany' },
+    economy: { GDP: 200 },
+    colors: ['Red', 'Yellow', 'Black'],
   },
 ];
 
 const properties = {
   name: { label: 'Name', search: true },
   'location.city': { label: 'City' },
   'economy.GDP': { label: 'GDP' },
+  colors: {
+    label: 'Flag Colors',
```

Comment:
```suggestion
    label: 'Flag colors',
```
sentence casing to align with DataTable column header/best practices.